modules/stats.php

```diff
@@ -1122,37 +1409,37 @@ function stats_print_wp_remote_error( $get, $url ) {
 	if ( $error !== $previous_error ) {
 ?>
 	<div class="wrap">
-	<p><?php _e( 'We were unable to get your stats just now. Please reload this page to try again.', 'jetpack' ); ?></p>
+	<p><?php esc_attr_e( 'We were unable to get your stats just now. Please reload this page to try again.', 'jetpack' ); ?></p>
 	</div>
 <?php
 		return;
 	}
 ?>
 	<div class="wrap">
-	<p><?php printf( __( 'We were unable to get your stats just now. Please reload this page to try again. If this error persists, please <a href="%1$s" target="_blank">contact support</a>. In your report please include the information below.', 'jetpack' ), 'http://support.wordpress.com/contact/?jetpack=needs-service' ); ?></p>
+	<p><?php printf( esc_attr_e( 'We were unable to get your stats just now. Please reload this page to try again. If this error persists, please <a href="%1$s" target="_blank">contact support</a>. In your report please include the information below.', 'jetpack' ), 'https://support.wordpress.com/contact/?jetpack=needs-service' ); ?></p>
```

Comment:
This is also wrong, this needs `esc_html__` so that `printf` is able to insert values into the translated string.